src/Security/Authorization/Core/src/AuthorizationServiceCollectionExtensions.cs

```diff
@@ -17,8 +17,8 @@ public static class AuthorizationServiceCollectionExtensions
     /// Adds authorization services to the specified <see cref="IServiceCollection" />.
     /// </summary>
     /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
-    /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
-    public static IServiceCollection AddAuthorizationCore(this IServiceCollection services)
+    /// <returns>The <see cref="AuthorizationBuilderServiceCollection"/> so that additional calls can be chained.</returns>
+    public static AuthorizationBuilderServiceCollection AddAuthorizationCore(this IServiceCollection services)
```

Comment:
This is a breaking change.

Reply:
Is this one we can get approved, functionally it shouldn't affect most people right?